source/quickstart/windows.rst

```diff
@@ -6,31 +6,115 @@ Quickstart for Windows
 
 .. default-domain:: minio
 
-This page deploys MinIO on Windows for use with early development
-and evaluation environments.
+.. |OS| replace:: Windows
 
-For instructions on deploying to production environments,
-see :ref:`deploy-minio-distributed`.
+Use the procedures on this page to deploy MinIO in :guilabel:`Standalone Mode` on a single local |OS| harddrive as a single MinIO server process (what we call "filesystem mode") with a :ref:`subset of S3 features<minio-installation-comparison>`.
+The MinIO server provides an S3 access layer to the drive or volume with no :ref:`erasure coding <minio-erasure-coding>` or associated features.
+
+Use :guilabel:`Standalone Mode` for early development and evaluation environments.
+For instructions on deploying to production environments, see :ref:`deploy-minio-distributed`.
 
 Prerequisites
 -------------
 
-ToDo
+- Read, write, and execute permissions for the user's directory
+- Familiarity with using the Command Prompt or PowerShell
 
 Procedure
 ---------
 
-#. Step 1
+#. Download the MinIO Server
+
+   Download the MinIO executable from the following URL:
+
+   .. code-block:: shell
+      :class: copyable
+
+      https://dl.min.io/server/minio/release/windows-amd64/minio.exe
+      
+   Do not attempt to run the executable.
+   You call the executable to launch the server.
+
+#. Launch the :mc:`minio server`
+
+   In PowerShell or the Command Prompt, navigate to the location of the executable or add the path of the ``minio.exe`` file to the system ``$PATH``.
+   
+   Use this command to start a local MinIO instance in the ``C:\minio`` folder.
+   If desired, you can replace ``C:\minio`` with another location to use for the MinIO instance.
+
+   .. code-block:: shell
+      :class: copyable
+
+      .\minio.exe server C:\minio
+
+   The :mc:`minio server` process prints its output to the system console, similar to the following:
+
+   .. code-block:: shell
+
+      API: http://192.0.2.10:9000  http://127.0.0.1:9000
+      RootUser: minioadmin
+      RootPass: minioadmin
+
+      Console: http://192.0.2.10:9001 http://127.0.0.1:9001
+      RootUser: minioadmin
+      RootPass: minioadmin
+
+      Command-line: https://docs.min.io/docs/minio-client-quickstart-guide
+         $ mc alias set myminio http://192.0.2.10:9000 minioadmin minioadmin
+
+      Documentation: https://docs.min.io
+
+      WARNING: Detected default credentials 'minioadmin:minioadmin', we recommend that you change these values with 'MINIO_ROOT_USER' and 'MINIO_ROOT_PASSWORD' environment variables.
+
```

Comment:
We should mention here the process is tied to the current PowerShell/CMD window. Closing the window stops the server.